css/doc.css

```diff
@@ -0,0 +1,106 @@
+.head-nav {
+    position: relative;
+    top: 0;
+    width: 100%;
+    padding: 3rem 10rem;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+header .logo {
+    position: relative;
+    color: var(--primary-color);
+    font-size: 3rem;
+    text-decoration: none;
+    text-transform: uppercase;
+    font-weight: 800;
+    letter-spacing: 1px;
+}
+
+header .navigation a {
+    color: var(--primary-color);
+    text-decoration: none;
+    font-weight: 500;
+    letter-spacing: 0.1rem;
+    padding: 3.2rem 1.5rem;
+}
+
+.doc-container {
+    width: 100%;
+    margin: 50px auto 0;
+    display: flex;
+    padding: 0 1rem;
+}
+
+.doc-content {
+    min-width: 0;
+    margin: 0px auto;
+    max-width: 1000px;
+    flex: 1 1 0%;
+}
```

Comment:
At some places you have used px and at some places rem  use same unit across the code for consistency . For  very small values like 0.1 or 0.0125 px can be used instead of rem. Rest looks good to me.

Reply:
okay i'll look into it,